src/pages/login.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+function Login() {
+  return (
+    <div>
+      <form>
+        <label htmlFor="fname">First name:</label>
```

Comment:
I think there's an ESLint error here, you can fix this by putting the <input /> inside the <label />, like this: 
![image](https://user-images.githubusercontent.com/54759887/203614060-92748fce-361d-4d94-8a2a-dd078484c494.png)

Reply:
Everything else looks good!